geomesa-accumulo/geomesa-accumulo-datastore/src/test/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStoreAlterSchemaTest.scala

```diff
@@ -8,235 +8,170 @@
-        features.sortBy(_.getID).map(_.getAttribute("geom").toString) mustEqual Seq("POINT (51 50)", "POINT (52 50)")
-        features.sortBy(_.getID).map(_.getAttribute("dtg"))
-            .map(d => ZonedDateTime.ofInstant(toInstant(d.asInstanceOf[Date]), ZoneOffset.UTC).getHour) mustEqual Seq(1, 2)
-      }
-      "for old attributes with new features" >> {
-        val query = new Query(sftName, ECQL.toFilter("IN ('f1')"), Array("geom", "dtg"))
-        val features = SelfClosingIterator(ds.getFeatureSource(sftName).getFeatures(query).features).toList
-        features.map(_.getID) must containTheSameElementsAs(Seq("f1"))
-        features.head.getAttribute("geom").toString mustEqual "POINT (51 50)"
-        ZonedDateTime.ofInstant(toInstant(features.head.getAttribute("dtg").asInstanceOf[Date]), ZoneOffset.UTC).getHour mustEqual 1
-      }
-      "for old attributes with old features" >> {
-        val query = new Query(sftName, ECQL.toFilter("IN ('f2')"), Array("geom", "dtg"))
-        val features = SelfClosingIterator(ds.getFeatureSource(sftName).getFeatures(query).features).toList
-        features.map(_.getID) must containTheSameElementsAs(Seq("f2"))
-        features.head.getAttribute("geom").toString mustEqual "POINT (52 50)"
-        ZonedDateTime.ofInstant(toInstant(features.head.getAttribute("dtg").asInstanceOf[Date]), ZoneOffset.UTC).getHour mustEqual 2
-      }
-      "for new attributes with new and old features" >> {
-        val query = new Query(sftName, ECQL.toFilter("IN ('f1', 'f2')"), Array("geom", "attr1"))
-        val features = SelfClosingIterator(ds.getFeatureSource(sftName).getFeatures(query).features).toList
-        features.map(_.getID) must containTheSameElementsAs(Seq("f1", "f2"))
-        features.sortBy(_.getID).map(_.getAttribute("geom").toString) mustEqual Seq("POINT (51 50)", "POINT (52 50)")
-        features.sortBy(_.getID).map(_.getAttribute("attr1")) mustEqual Seq("1", null)
-      }
-      "for new attributes with new features" >> {
-        val query = new Query(sftName, ECQL.toFilter("IN ('f1')"), Array("geom", "attr1"))
-        val features = SelfClosingIterator(ds.getFeatureSource(sftName).getFeatures(query).features).toList
-        features.map(_.getID) must containTheSameElementsAs(Seq("f1"))
-        features.head.getAttribute("geom").toString mustEqual "POINT (51 50)"
-        features.head.getAttribute("attr1") mustEqual "1"
-      }
-      "for new attributes with old features" >> {
-        val query = new Query(sftName, ECQL.toFilter("IN ('f2')"), Array("geom", "attr1"))
-        val features = SelfClosingIterator(ds.getFeatureSource(sftName).getFeatures(query).features).toList
-        features.map(_.getID) must containTheSameElementsAs(Seq("f2"))
-        features.head.getAttribute("geom").toString mustEqual "POINT (52 50)"
-        features.head.getAttribute("attr1") must beNull
-      }
-    }
-    "update schemas" in {
```

Comment:
Just trying to follow the bits...  Did this "update schemas" test case make the jump?

Reply:
yeah consolidated them into AccumuloDataStoreAlterSchemaTest